packages/backend/src/accounting/service.ts

```diff
@@ -85,10 +70,12 @@ interface AssetAccountOptions {
     unit: number
     account: AssetAccount
   }
-  withBalance?: never
 }
 
-export type AccountOptions = BasicAccountOptions | AssetAccountOptions
+export type AccountOptions = (BasicAccountOptions | AssetAccountOptions) & {
+  receivedAccountId?: string
+  sentAccountId?: string
+}
```

Comment:
It looks like this comment is on an outdated commit. Here's the final form:
https://github.com/interledger/rafiki/pull/197/files#diff-72a5530c2c89f105f75e071479a382e8f8408459d37968fb93b546cca4783b31R75-R91
There's currently not a situation in which an account would have both a received and sent account id.
With this previous iteration, it could have been consolidated to one account whose type could be checked, but I like how it fit into separating the connector's incoming and outgoing account types.
https://github.com/interledger/rafiki/pull/197/files#diff-08011ba8622ef9b1a38af778014f9471eb8ce942adc8609074f019765f7338f6R26-R43